client/webserver/site/src/js/locales.ts

```diff
@@ -78,6 +78,23 @@ export const ID_ARCHIVED_RECORDS_PATH = 'ARCHIVED_RECORDS_PATH'
 export const ID_DEFAULT = 'DEFAULT'
 export const ID_ADDED = 'USER_ADDED'
 export const ID_DISCOVERED = 'DISCOVERED'
+export const ID_UNSUPPORTED_ASSET_INFO_ERR_MSG = 'UNSUPPORTED_ASSET_INFO_ERR_MSG'
+export const ID_LIMIT_ORDER = 'LIMIT_ORDER'
+export const ID_LIMIT_ORDER_IMMEDIATE_TIF = 'LIMIT_ORDER_IMMEDIATE_TIF'
+export const ID_MARKET_ORDER = 'MARKET_ORDER'
+export const ID_MATCH_STATUS_NEWLY_MATCHED = 'MATCH_STATUS_NEWLY_MATCHED'
+export const ID_MATCH_STATUS_MAKER_SWAP_CAST = 'MATCH_STATUS_MAKER_SWAP_CAST'
+export const ID_MATCH_STATUS_TAKER_SWAP_CAST = 'MATCH_STATUS_TAKER_SWAP_CAST'
+export const ID_MATCH_STATUS_MAKER_REDEEMED = 'MATCH_STATUS_MAKER_REDEEMED'
+export const ID_MATCH_STATUS_REDEMPTION_SENT = 'MATCH_STATUS_REDEMPTION_SENT'
+export const ID_MATCH_STATUS_MATCH_COMPLETE = 'MATCH_STATUS_MATCH_COMPLETE'
```

Comment:
`ID_MATCH_STATUS_MATCH_COMPLETE` does not have an entry below. It also looks unused in the codebase.

Reply:
But there is also `ID_MATCH_STATUS_COMPLETE` so maybe only need one or the other?